app/presenters/email_alert_presenter.rb

```diff
@@ -24,6 +25,19 @@ def body
     end
   end
 
+  def tags
+    { format: document.publishing_api_document_type }.deep_merge(metadata)
+  end
+
+  def metadata
```

Comment:
I find this method a bit cryptic. Do the fields have to be dynamic here, or are only a few used (eg `format`, `case_type`)?